.dockerignore

```diff
@@ -1,10 +1,9 @@
 **/node_modules
 **/dist
 **/tmp
+**/docs
 
-docs
 infrastructure
-localenv
```

Comment:
ah, did this end up breaking `localenv` 😅

Reply:
yep 😅